package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-components",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-components/issues",
-  "version": "8.7.0",
+  "version": "9.0.0-beta.0",
```

Comment:
is it breaking?

Reply:
https://github.com/good-hood-gmbh/nebenan-frontend/pull/1007#discussion_r420093265

> Major version update because DateBox could break after update because css is quite different (now uses flex, previously only used paddings). css overrides may don't work anymore

(and I removed a prop -> major update)